src/components/shared/nav-bar/nav-bar.module.scss

```diff
@@ -1,19 +1,40 @@
 @use '@variables' as var;
 @use '@mixins' as mixins;
 
-.navigation {
+.container {
   position: sticky;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   top: 0;
+  @include mixins.pagePadding;
+  width: 100%;
+  background-color: var.$white;
+  z-index: 100;
+}
+
+.navigation {
   display: flex;
   align-items: center;
   justify-content: space-between;
-  margin: 0 auto;
   width: 100%;
   max-width: 1131px;
   height: 120px;
-  z-index: 100;
-  @include mixins.pagePadding;
-  background-color: var.$white;
+}
+
+.background {
```

Comment:
can you make this classname more specific? background of what, is it the entire nav?